### DISCUSSION TITLE
Making Maps in R

### DISCUSSION THREAD

**Discussion by ChrisMantegna** [RobertsLab/resources#1586] [Show and tell]
Woohoo!
Using code from: https://eriqande.github.io/rep-res-web/lectures/making-maps-with-R.html
I was able to make maps in R. I put the relevant steps together for what I needed in a rmd file called mapMaking. The repo has the files and code: https://github.com/ChrisMantegna/WDFWmussels

**Comment by AHuffmyer** [comment]
The top part of the script using the R data worked great for me! Thanks so much Chris!
